source/virtualBuffers/MSHTML.py

```diff
@@ -264,14 +264,23 @@ def _searchableAttribsForNodeType(self,nodeType):
 		elif nodeType=="formField":
 			attrs=[
 				{"IAccessible::role":[oleacc.ROLE_SYSTEM_PUSHBUTTON,oleacc.ROLE_SYSTEM_RADIOBUTTON,oleacc.ROLE_SYSTEM_CHECKBUTTON,oleacc.ROLE_SYSTEM_OUTLINE],"IAccessible::state_%s"%oleacc.STATE_SYSTEM_READONLY:[None]},
-				{"IAccessible::role":[oleacc.ROLE_SYSTEM_COMBOBOX,oleacc.ROLE_SYSTEM_TEXT]},
+				{"IAccessible::role":[oleacc.ROLE_SYSTEM_COMBOBOX]},
+				# Focusable edit fields (input type=text, including readonly ones)
+				{"IAccessible::role":[oleacc.ROLE_SYSTEM_TEXT],"IAccessible::state_%s"%oleacc.STATE_SYSTEM_FOCUSABLE:[1]},

```

Comment:
As discussed I found this a little confusing to read. Specifically that `STATE_SYTEM_FOCUSABLE` belongs  to the format string, and is the key to the value `[1]`. You could consider doing `"IAccessible::state_{}".format(oleacc.STATE_SYSTEM_FOCUSABLE) : [1]` instead.